src/differentials/abstract_zero.jl

```diff
@@ -26,14 +58,8 @@ arguments.
     end
 ```
 """
-struct DoesNotExist <: AbstractDifferential end
+struct DoesNotExist <: AbstractZero end
```

Comment:
Docstring needs updating to make clear it is a subtype of AbstractZero, potentially with a note on why

Reply:
We have already got:
```
...
The only valid way to pertube such a values is to not change it at all.
As such, `DoesNotExist` is functionally identical to `Zero()`,
```